examples/billing/get_billing_setup.py

```diff
@@ -37,47 +35,45 @@ def main(client, customer_id, page_size):
         'billing_setup.payments_account_info.secondary_payments_profile_id '
         'FROM billing_setup')
 
-    results = ga_service.search(customer_id, query=query, page_size=page_size)
+    response = ga_service.search_stream(customer_id, query=query)
 
     try:
         # Use the enum type to determine the enum name from the value.
-        billing_setup_status_enum = (
-            client.get_type('BillingSetupStatusEnum', version='v3')
-                .BillingSetupStatus)
+        billing_setup_status_enum = client.get_type(
+            'BillingSetupStatusEnum', version='v3').BillingSetupStatus
 
         print('Found the following billing setup results:')
-        for row in results:
-            billing_setup = row.billing_setup
-            payments_account_info = billing_setup.payments_account_info
-            print('Billing setup with ID "%s", status "%s", '
-                  'payments_account "%s", payments_account_id "%s", '
-                  'payments_account_name "%s", payments_profile_id "%s", '
-                  'payments_profile_name "%s", '
-                  'secondary_payments_profile_id "%s".'
-                  % (billing_setup.id.value,
-                     billing_setup_status_enum.Name(billing_setup.status),
-                     billing_setup.payments_account.value,
-                     payments_account_info.payments_account_id.value,
-                     payments_account_info.payments_account_name.value,
-                     payments_account_info.payments_profile_id.value,
-                     payments_account_info.payments_profile_name.value,
-                     payments_account_info.secondary_payments_profile_id.value))
-    except google.ads.google_ads.errors.GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
+        for batch in response:
+            for row in batch.results:
+                billing_setup = row.billing_setup
+                pai = billing_setup.payments_account_info
+                if pai.secondary_payments_profile_id.value:
+                    secondary_payments_profile_id = pai.secondary_payments_profile_id.value
+                else:
+                    secondary_payments_profile_id = "None"
```

Comment:
Can't this be just:

```python
secondary_payments_profile_id = (pai.secondary_payments_profile_id.value 
                                 if pai.secondary_payments_profile_id.value
                                 else "None")
```

?

Reply:
It can be, but I'm not a big fan of that syntax as it hides intentionality somewhat. Since the rest of this example does not use it, I didn't want to introduce it.